app/services/person_with_remote_fields.rb

```diff
@@ -0,0 +1,49 @@
+class PersonWithRemoteFields < SimpleDelegator
+  REMOTE_FIELDS = [
+    :email, :phone, :first_name, :last_name, :is_volunteer,
+    :employer, :occupation, :skills, :tags
+  ]
+  LOCAL_FIELDS = [
+    :email, :phone, :first_name, :last_name,
+    :address, :city, :zip, :is_volunteer
+  ]
+
+  def self.find_or_build(attributes)
+    email = attributes.symbolize_keys.delete(:email)
+    person = Person.find_or_initialize_by(email: email)
```

Comment:
This is actually pretty important when we're doing telephony work.  You wrote and then I updated the `create_or_update` on `Person`.  Let's use that.